mods/rv/rules/default-structures.yaml

```diff
@@ -1,7 +1,16 @@
+^AISellWhenMindControlled:
+	IssueOrderToBot@AISellOnMC:
+		RequiresCondition: controlled
+		OrderName: Sell
+		OrderTrigger: Periodically
+		OrderChance: 100
+		OrderInterval: 45
+
 ^ConstructionYard:
 	Inherits: ^BaseBuilding
 	Inherits@EMPULSE: ^DisabledByEMP
 	Inherits@MC: ^MindControllableBuilding
+	Inherits@AISell: ^AISellWhenMindControlled
```

Comment:
It seems like you put this on all the base buildings, can't it just go to `^MindControllableBuilding`? Or am i missing something that is MCable but shouldn't be sold?

Reply:
Tech buildings